[2023-06-24T10:30:57.660+0000] {processor.py:153} INFO - Started process (PID=60) to work on /opt/airflow/dags/spark_jobs_cluster_dag.py
[2023-06-24T10:30:57.661+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/spark_jobs_cluster_dag.py for tasks to queue
[2023-06-24T10:30:57.661+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:30:57.661+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/spark_jobs_cluster_dag.py
[2023-06-24T10:30:57.662+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/spark_jobs_cluster_dag.py
[2023-06-24T10:30:57.678+0000] {logging_mixin.py:137} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-06-24T10:30:57.686+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:30:57.686+0000] {taskinstance.py:1777} ERROR - {'DAG Id': 'spark_airflow_dag_cluster_v03', 'Task Id': 'spark_submit_task', 'Run Id': 'manual__2023-06-21T17:14:20.678202+00:00', 'Hostname': '176a044b0668', 'External Executor Id': '16703c76-8eae-453f-a488-d579542309c8'}
[2023-06-24T10:30:57.691+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:30:57.691+0000] {taskinstance.py:1321} INFO - Marking task as FAILED. dag_id=spark_airflow_dag_cluster_v03, task_id=spark_submit_task, execution_date=20230621T171420, start_date=20230621T171421, end_date=20230624T103057
[2023-06-24T10:30:57.699+0000] {processor.py:713} INFO - Executed failure callback for <TaskInstance: spark_airflow_dag_cluster_v03.spark_submit_task manual__2023-06-21T17:14:20.678202+00:00 [failed]> in state failed
[2023-06-24T10:30:57.700+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/spark_jobs_cluster_dag.py took 0.042 seconds
[2023-06-24T10:31:07.747+0000] {processor.py:153} INFO - Started process (PID=68) to work on /opt/airflow/dags/spark_jobs_cluster_dag.py
[2023-06-24T10:31:07.748+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/spark_jobs_cluster_dag.py for tasks to queue
[2023-06-24T10:31:07.748+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:31:07.748+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/spark_jobs_cluster_dag.py
[2023-06-24T10:31:07.748+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/spark_jobs_cluster_dag.py
[2023-06-24T10:31:07.764+0000] {logging_mixin.py:137} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-06-24T10:31:07.771+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:31:07.771+0000] {taskinstance.py:1777} ERROR - {'DAG Id': 'spark_airflow_dag_cluster', 'Task Id': 'spark_submit_task', 'Run Id': 'manual__2023-06-21T16:41:40.332826+00:00', 'Hostname': '176a044b0668', 'External Executor Id': '66c938c3-3cdf-4bb1-ba51-1598e72be3c9'}
[2023-06-24T10:31:07.776+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:31:07.775+0000] {taskinstance.py:1321} INFO - Marking task as FAILED. dag_id=spark_airflow_dag_cluster, task_id=spark_submit_task, execution_date=20230621T164140, start_date=20230621T164140, end_date=20230624T103107
[2023-06-24T10:31:07.781+0000] {processor.py:713} INFO - Executed failure callback for <TaskInstance: spark_airflow_dag_cluster.spark_submit_task manual__2023-06-21T16:41:40.332826+00:00 [failed]> in state failed
[2023-06-24T10:31:07.783+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:31:07.783+0000] {taskinstance.py:1777} ERROR - {'DAG Id': 'spark_airflow_dag_cluster', 'Task Id': 'spark_submit_task', 'Run Id': 'manual__2023-06-21T16:41:40.332826+00:00', 'Hostname': '176a044b0668', 'External Executor Id': '66c938c3-3cdf-4bb1-ba51-1598e72be3c9'}
[2023-06-24T10:31:07.786+0000] {logging_mixin.py:137} INFO - [2023-06-24T10:31:07.785+0000] {taskinstance.py:1321} INFO - Marking task as FAILED. dag_id=spark_airflow_dag_cluster, task_id=spark_submit_task, execution_date=20230621T164140, start_date=20230621T164140, end_date=20230624T103107
[2023-06-24T10:31:07.788+0000] {processor.py:713} INFO - Executed failure callback for <TaskInstance: spark_airflow_dag_cluster.spark_submit_task manual__2023-06-21T16:41:40.332826+00:00 [failed]> in state failed
[2023-06-24T10:31:07.789+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/spark_jobs_cluster_dag.py took 0.044 seconds
